Homework_QAA/mongo_DB/mongo_db.py

```diff
@@ -0,0 +1,61 @@
+import pymongo
+
+class BaseDB:
+    def __init__(self, host: str, port: int, database: str):
+        self._host = host
+        self._port = port
+        self._database = database
+        self._client = pymongo.MongoClient(f'{self._host}:{self._port}/?directConnection=true&serverSelectionTimeoutMS=2000&appName=mongosh+1.6.1')
+        self._db = self._client[self._database]
+        self._collection = self._db['test_team']
+
+    def insert_one(self, key: str, value: str):
```

Comment:
just dict should be as a parameter. What if I want to add smth which is contains more than one key:value pairs